### DISCUSSION TITLE
Ability to group findings

### DISCUSSION THREAD

**Discussion by MichaelGrafnetter** [Syslifters/sysreptor#308] [General]
When writing pentest reports, I usually split the findings into chapters, based on the application layer, server, service, etc.  Each document section/chapter typically starts with an Overview/Introduction text. I wonder if such grouping could be achieved with SysReptor. The Findings view could have a hierarchical (at least 1 level) structure instead of a flat one as well:

![image](https://github.com/user-attachments/assets/f2e0cabc-14fe-488c-bab0-1bab8f3bbcc1)
 
This would especially be useful if there are hundreds of findings to be reported.

**Comment by MWedl** [comment]
Hi,
it is already possible to group findings into separate chapters in the PDF report by grouping them via VueJS template language during PDF rendering. See #90 and #4 for examples.

The finding sidebar currently only supports a flat list. However, you can customize the finding ordering (see #69) to sort findings by a category field (e.g. enum or combobox with application layer, server, service, etc.), then by CVSS. By ordering via a category field first, findings of the same category appear next to each other in the list. There are no group headlines, though.

**Comment by MichaelGrafnetter** [comment]
Thanks @MWedl . Yes, for standardized reports, your answer would be a perfect solution. But for one-off, report-specific section headings, creating a custom report template and field would cause too much overhead.

**Comment by aronmolnar** [comment]
Grouping findings is now officially supported, also in the UI https://docs.sysreptor.com/designer/findings/#finding-groups
